sabnzbd/newswrapper.py

```diff
@@ -276,11 +276,17 @@ def __init__(self, nw: NewsWrapper, host):
         if is_ipv6_addr(self.host):
             af = socket.AF_INET6
 
+        sabnzbd.misc.set_socks5_proxy()
+        self.sock = socket.socket(af, socktype, proto)
+        if sabnzbd.cfg.socks5_proxy_url():
+            try:
+                self.sock.connect((self.host, self.nw.server.port))
```

Comment:
Why does this one have to be seperate from the regular `socket.connect`?

Reply:
It has to connect before adding the SSL layer, otherwise it doesn't work. I don't know why but there doesn't seem to be any way around it.